web/client/selectors/contextcreator.js

```diff
@@ -50,7 +50,14 @@ export const prefetchedDataSelector = state => get(state, 'contextcreator.prefet
 export const disableImportSelector = state => creationStepSelector(state) !== "general-settings";
 
 export const generateContextResource = (state) => {
-    const mapConfig = mapSelector(state) ? mapSaveSelector(state) : {};
+    // in some cases: mapSelector(state) includes only eventListeners so check if mapSelector includes center prop
+    const isMapConfigNotChanged = !mapSelector(state)?.center;
+    let mapConfig = {};
+    if (isMapConfigNotChanged) {
+        mapConfig = mapConfigSelector(state) || {};
+    } else {
+        mapConfig = mapSelector(state) ? mapSaveSelector(state) : {};
+    }
```

Comment:
```suggestion
    const map = mapSelector(state);
    const mapConfig = map ? map?.center ? mapSaveSelector(state) : mapConfigSelector(state) : {};
```

How about something like this?

Reply:
@dsuren1 
for me it is Ok, but some reviewers prefer the if approach as it eases the debug/review step

Reply:
In that case, mapSelector is called twice which is not needed and `isMapConfigNotChanged` is confusing maybe `isMapConfigPresent`